src/index.js

```diff
@@ -12,7 +12,7 @@
 // of values or Errors.
 type BatchLoadFn<K, V> = (keys: Array<K>) => Promise<Array<V | Error>>;
 
-// Optionally turn off batching or caching or provide a cache key function or a
+// Optionally turn off batching or caching, and provide a cache key function or
```

Comment:
Strictly speaking, this change makes things worse. Given that all the properties are optional, "or" is the appropriate conjunction here. So I'll revert this hunk before merging. (To be honest, this comment has close to zero value anyway because it just reformulates what's encoded in the type using sloppier language. I wouldn't be averse to just deleting it.)